readme.txt

```diff
@@ -82,6 +84,7 @@ Easy Digital Downloads is a verified member of the [Stripe partner program](http
 
 With add-on plugins from our [extensions catalogue](https://easydigitaldownloads.com/downloads/?utm_source=extensions&utm_medium=description_tab&utm_content=extensions&utm_campaign=readme) you can super-charge your digital store. Some of our extension highlights include:
 
+* [Stripe Pro](https://easydigitaldownloads.com/downloads/stripe-gateway/?utm_source=extension&utm_medium=description_tab&utm_content=software-licensing&utm_campaign=readme) removes the additional 2% fee we charge as well as adds the ability to accept pre-approved payments (extension must be installed and activated to remove the additional 2% fee)
```

Comment:
@pderksen does this need to say "we charge" as it's not really us charging; it's an additional fee collected by Stripe? Also the parenthetical ends up feeling a bit redundant. Maybe simplify to this instead?

>Stripe Pro removes the additional 2% processing fee as well as adds the ability to accept pre-approved payments (extension must be installed and activated)

Reply:
@robincornett Yes much better wording. Approved!